app/seed-data/emissions_factors/scripts/EFDB_2006_IPCC_guidelines_Stationary_Energy.py

```diff
@@ -920,9 +969,18 @@ def expand_metadata(metadata):
     }
     EF_df_fugitive['parameters'] = EF_df_fugitive['parameters'].replace(mapping_parameters)
 
+    EF_df_fugitive['extra'] = EF_df_fugitive['extra'].replace({
+        'undeground mines': 'underground mines', 
+        'undeground mining': 'underground mines', 
+        'underground mining': 'underground mines',
+        'undeground post-mining': 'underground post-mining'
```

Comment:
I think the values in this dict weren't updated - you're replacing all values by themselves?

Reply:
I'm not replacing them with themselves, I'm assigning the same string, there are some "r" and changing mining by mines